### DISCUSSION TITLE
Opening Terminal Problem

### DISCUSSION THREAD

**Discussion by Ademhackergity** [community/community#28269] [Codespaces]
Hi I learning CS50 from Harvard university and I can't open the terminal in the web because it saying  You cannot create codespaces with this repository so what can I do please help as soon as possible because I want to continue learning the course and thanks.

**Comment by KartikM123** [comment]
Hi @Ademhackergity ! Thanks for raising this issue!

From the error message, I'm guessing you're trying to create a codespace on the CS50 repo? If so, can you walk me through how you're trying to create a codespace?

**Comment by jfullstackdev** [comment]
also, I don't know whether there is an authorization but I'm seeing a lot of posts like this in the past
